docs/training.md

```diff
@@ -0,0 +1,204 @@
+# Training
+
+The prebuilt models will always be improved by adding data from the target area. In our work, we have found that even one hour's worth of carefully chosen hand-annotation can yield enormous improvements in accuracy and precision.
+We envision that for the majority of scientific applications at least some fine-tuning of the prebuilt model will be worthwhile. When starting from the prebuilt model for training, we have found that 5-10 epochs is sufficient. 
+We have never seen a retraining task that improved after 10-30 epochs, but it is possible if there are very large datasets with very diverse classes.
+
+Consider an annotations.csv file in the following format
+
+testfile_deepforest.csv
+
+```
+image_path, xmin, ymin, xmax, ymax, label
+OSBS_029.jpg,256,99,288,140,Tree
+OSBS_029.jpg,166,253,225,304,Tree
+OSBS_029.jpg,365,2,400,27,Tree
+OSBS_029.jpg,312,13,349,47,Tree
+OSBS_029.jpg,365,21,400,70,Tree
+OSBS_029.jpg,278,1,312,37,Tree
+OSBS_029.jpg,364,204,400,246,Tree
+OSBS_029.jpg,90,117,121,145,Tree
+OSBS_029.jpg,115,109,150,152,Tree
+OSBS_029.jpg,161,155,199,191,Tree
+```
+
+We tell the config that we want to train on this csv file, and that the images are in the same directory. If images are in a separate folder, change the root_dir.
```

Comment:
The config file specifies the path to the CSV file that we want to use when training. The images are located in the working directory by default, and a user can provide a path to a different image directory.